src/workflows-app/FeaturedWorkflows.ts

```diff
@@ -112,118 +202,57 @@ export const FeaturedWorkflows = ({
-              );
-              return [
-                matchingMethod !== undefined,
-                { ...featuredMethod, last_run: matchingMethod ? matchingMethod.last_run : featuredMethod.last_run },
-              ] as const;
-            });
-            const allMethodsInWorkspace = methodsInWorkspace.every(([inWorkspace, _method]) => inWorkspace);
-            const noMethodsInWorkspace = methodsInWorkspace.every(([inWorkspace, _method]) => !inWorkspace);
-            const newMethodData = methodsInWorkspace.map(([_inWorkspace, method]) => method);
-            return h(
-              WorkflowCard,
-              {
-                key: method.name,
-                method: 'methods' in method ? { ...method, methods: newMethodData } : newMethodData[0],
-              },
-              [
-                allMethodsInWorkspace
-                  ? div(
-                      {
-                        style: {
-                          display: 'flex',
-                          alignItems: 'center',
-                          justifyContent: 'center',
-                          borderRadius: 2,
-                          color: colors.light(0.5),
-                          fontWeight: 500,
-                          padding: '0.75rem 0',
-                          whiteSpace: 'pre',
-                          backgroundColor: colors.success(1),
-                        },
-                      },
-                      [icon('success-standard'), '   Added']
-                    )
-                  : div({ style: { display: 'flex', alignItems: 'center', flexDirection: 'row' } }, [
-                      !noMethodsInWorkspace &&
-                        h(
-                          TooltipTrigger,
-                          {
-                            useTooltipAsLabel: true,
-                            content:
-                              'You already have some of the workflows in this set added to this workspace. Clicking add to workspace will only add the remaining workflows.',
-                          },
-                          [icon('info-circle', { style: { marginRight: '1rem' }, size: 16, color: colors.accent(1) })]
-                        ),
+  const renderFeaturedWorkflows = () =>
+    h(Fragment, [
+      div(['Get up and running with these commonly used, standard workflows.']),
+      div({ style: { marginTop: '1rem' } }, [
+        _.map((method) => {
+          const methodsInWorkspace = getMethodsInWorkspace(method);
+          const allMethodsInWorkspace = methodsInWorkspace.every(([inWorkspace, _method]) => inWorkspace);
+          return h(
+            WorkflowCard,
+            {
+              key: method.name,
+              method,
+            },
+            [allMethodsInWorkspace ? addedButton() : addToWorkspaceButton(method, methodsInWorkspace)]
+          );
+        }, featuredWarpWorkflows as FeaturedWorkflow[]),
+      ]),
+      div(
+        { style: { marginTop: '1rem' } },
```

Comment:
Not sure if we need this margin on the top, there's a bit of a weird gap currently between the warp workflows and the covid workflows. If they are all mapped over as one object it should also remove any irregularities.